gunicorn/config.py

```diff
@@ -346,6 +346,15 @@ def validate_pos_int(val):
     return val
 
 
+def validate_float(val):
+    if not val:
+        return 0.0
```

Comment:
Or keep it closer to what we have and just do `0.1` as the default and use `* random.random()` like before. Will that not work?